cmd/egress/cmd.go

```diff
@@ -275,6 +273,7 @@ are set correctly before execution.
 		validateEgressCmd.PrintErr(err)
 	}
 
+	validateEgressCmd.MarkFlagsMutuallyExclusive("cacert", "no-tls")
```

Comment:
Now that we have this, the check and log on line 94-96 should be removed, as it is not possible to hit that codepath.

Reply:
i already removed that check